files/build_templates/sonic_debian_extension.j2

```diff
@@ -233,7 +233,7 @@ sudo cp -f $IMAGE_CONFIGS/bash/bash.bashrc $FILESYSTEM_ROOT/etc/
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install libcairo2-dev libdbus-1-dev libgirepository1.0-dev libsystemd-dev pkg-config
 
 # Mark runtime dependencies as manually installed to avoid them being auto-removed while uninstalling build dependencies
-sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-mark manual gir1.2-glib-2.0 libdbus-1-3 libgirepository-1.0-1 libsystemd0
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-mark manual gir1.2-glib-2.0 libdbus-1-3 libgirepository-1.0-1 libsystemd0 python3-dbus
```

Comment:
>apt-mark manual [](http://example.com/codeflow?start=67&length=15)

This line is to "Mark runtime dependencies as manually installed", not to install a package.

Reply:
This change can avoid python3-dbus being auto-removed.
I have check the build log, below command will install python3-dbus:
sudo LANG=C chroot ./fsroot-vs apt-get -y install apt-transport-https ca-certificates curl gnupg2 software-properties-common
And below command will remove python3-dbus:
+ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot ./fsroot-vs apt-get autoremove -y